README.md

```diff
@@ -1 +1,114 @@
+  - [Getting Started with VueJS](https://medium.freecodecamp.org/a-quick-introduction-to-vue-js-72937ee8880d)
+  - [Making a ToDo application with VueJS](https://scotch.io/tutorials/build-a-to-do-app-with-vue-js-2)
+  - [Getting started with VueJS, apollo and Graphql](https://www.howtographql.com/vue-apollo/0-introduction/)
+  - [(Video) Using GraphQL, VueJS and Apollo to Create Blog Application](https://youtu.be/N49R1gifUqs)
+  - [(Video) VueJS, Apollo and GraphQL](https://www.howtographql.com/vue-apollo/0-introduction/)
+- Miscellaneous
+  - [Why VueJs and GraphQL can be ultimate stack](https://youtu.be/nq5q_gOOimA)
+
+## Boilerplates and Example Apps
+
+> Boilerplates and examples for a headstart in development
+
+- VueJS Apollo
+  - [Vue + Graphql + Apollo Tutorial](https://github.com/graphql-boilerplates/vue-fullstack-graphql)
+  - [VueJs Starter app with AWS amplify](https://github.com/aws-samples/aws-amplify-vue)
+  - [Hello World Vue Apollo Boilerplate](https://github.com/pradeepgangwar/Vue-Graphql/tree/master/hello-world)
+  - [Basic Vue Apollo Boilerplate (Todo App with basic auth)](https://github.com/pradeepgangwar/Vue-Graphql/tree/master/hello-world)
+
+## Advanced Use Cases
+
+- Server-side Rendering
+  - [Understanding Server-side rendering with Vue Apollo](https://akryum.github.io/vue-apollo/guide/ssr.html)
+- Realtime
+  - [Polling](https://dev-blog.apollodata.com/dynamic-graphql-polling-with-react-and-apollo-client-fb36e390d250): Dynamic GraphQL polling with React and Apollo Client
+  - [Subscriptions with Vue Apollo Client](https://akryum.github.io/vue-apollo/guide/apollo/subscriptions.html#setup): Learn how to achieve realtime data with GraphQL subscriptions using Vue Apollo Client.
+  - [Managing Local Data](https://github.com/apollographql/apollo-link-state): if You need to manage your app lcoal state you can do so with apollo link state.
+
+## GraphQL BaaS Providers
+
+> Get a GraphQL backend for your applications using these service providers
+
+- [Graphcool](https://www.graph.cool): Self-Hosted GraphQL BaaS
+- [Hasura](https://hasura.io): Instant GraphQL APIs on Postgres
+- [Scaphold](https://scaphold.io): GraphQL Backend As A Service
+- [AWS AppSync](https://aws.amazon.com/appsync/): Build data-driven apps with real-time and offline capabilities
+
+## Articles and Videos
+
+### Articles
+
+- [So what’s this GraphQL thing I keep hearing about?](https://medium.freecodecamp.org/so-whats-this-graphql-thing-i-keep-hearing-about-baf4d36c20cf)
+- [Thinking in Graphs](http://graphql.org/learn/thinking-in-graphs/)
+- [What are the benefits of using GraphQL and why it surpasses REST](https://blog.hellocomet.co/graphql-benefits-api/)
+
+### Videos
+
+- [From REST to GraphQL](https://www.youtube.com/watch?v=ntBU5UXGbM8)
+
+## Stories
+
+> Good and bad experiences
+
+- [The Evolution of React and GraphQL at Facebook and Beyond](https://developers.facebook.com/videos/f8-2017/the-evolution-of-react-and-graphql-at-facebook-and-beyond/)
```

Comment:
Article focusses on React. Apart from that, can we add more Vue.js related articles and videos before merging?